_posts/2021-04-24-exploring-a-spring-boot-app-with-actuator-and-jq.md

```diff
@@ -0,0 +1,826 @@
+---
+title: Exploring a Spring Boot App with Actuator and jq
```

Comment:
Should we change it to "Exploring a Spring Boot App with Actuator and jq JSON processor"?